src/apps/mpx.erl

```diff
@@ -112,7 +116,7 @@ handle_call({client_disconnected, #de_client{meta=Apps} = Client}, _, State) ->
 handle_call({handle_client_message, #de_client{data=AppData}=Client, App, Message}, _, State) ->
```

Comment:
Line 116 is too long: handle_call({handle_client_message, #de_client{data=AppData}=Client, App, Message}, _, State) ->.